examples/hello-world/.vscode/tasks.json

```diff
@@ -0,0 +1,29 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Watch and Compile Project",
+      "type": "shell",
+      "command": "npm",
+      "args": ["--silent", "run", "build:watch"],
+      "group": {
+        "kind": "build",
+        "isDefault": true
+      },
+      "problemMatcher": "$tsc-watch"
+    },
+    {
+      "label": "Build, Test and Lint",
+      "type": "shell",
+      "command": "npm",
+      "args": ["--silent", "run", "test:dev"],
```

Comment:
Shouldn't this be `test`?

Reply:
The 'advocated' development pattern according to `DEVELOPING.md` was to have `build:watch` running and then running the tests _without_ building (since `build:watch` takes care of that). `test:dev` is just `npm t` without the build step

Reply:
Ah ok. Makes sense. 👍